test/shared/helpers/forEach.test.js

```diff
@@ -103,11 +103,11 @@ describe('forEach', function () {
       var thisCall = spy.getCall(0),
           args = thisCall.args[0];
 
-      expect(args).to.deep.property('key', 0);
```

Comment:
You probably just needed to change these to `.to.have.property('key', value)`. These matchers will give you a much better failure message if you can figure out how to make them work.